front/pages/w/[wId]/ds/index.tsx

```diff
@@ -69,7 +74,7 @@ export const getServerSideProps: GetServerSideProps<{
   owner: WorkspaceType;
   readOnly: boolean;
   dataSources: DataSourceType[];
-  managedDataSources: ManagedDataSource[];
+  managedDataSources: ConnectorSupported[];
```

Comment:
nit: variable name is outdated

Reply:
my comment was about having the variable name match the type name, but think `ConnectorSupported` is not very explicit as a type name

Reply:
(yes obviously needs to be aligned)

Reply:
My bad. For me `managedDataSources` was very confusing when modifying that code because a dataSource record from the database that is managed is also commonly called a "managed data source".


I checked the AirByte source code, they call that "INTEGRATIONS", let's do the same.

Reply:
And sorry to be a pain about that @fontanierh and @spolu but it was a little torture for my little brain yesterday because in this file we have : 
```
const MANAGED_DATA_SOURCES: ManagedDataSource[] = [....
```

and: 
```
 let allDataSources = await getDataSources(auth);
  const dataSources = allDataSources.filter((ds) => !ds.connector);
  const managedDataSources = allDataSources.filter((ds) => ds.connector); <<<<<<<<<<<<<<<<<<<
```

Plus the `connectorStatuses`, it was hard to map everything in my mind and it took me forever to get it right. So I am assuming the next engineer behind me might struggle too.